common/src/leap/soledad/common/tests/test_couch.py

```diff
@@ -1039,18 +951,18 @@ def test_sync_sees_remote_conflicted(self):
         self.assertTransactionLog([doc_id, doc_id], self.db1)
         self.assertGetDoc(self.db1, doc_id, doc2_rev, new_doc, True)
         self.assertGetDoc(self.db2, doc_id, doc2_rev, new_doc, False)
-        from_idx = self.db1.get_from_index('test-idx', 'altval')[0]
-        self.assertEqual(doc2.doc_id, from_idx.doc_id)
-        self.assertEqual(doc2.rev, from_idx.rev)
-        self.assertTrue(from_idx.has_conflicts)
-        self.assertEqual([], self.db1.get_from_index('test-idx', 'value'))
+        # soledad doesnt support index due to encryption
+        # from_idx = self.db1.get_from_index('test-idx', 'altval')[0]
+        # self.assertEqual(doc2.doc_id, from_idx.doc_id)
+        # self.assertEqual(doc2.rev, from_idx.rev)
+        # self.assertTrue(from_idx.has_conflicts)
+        # self.assertEqual([], self.db1.get_from_index('test-idx', 'value'))
```

Comment:
Could we do something with this commented code? I have some kind of phobia to commented code. I would prefer to remove the code if it don't make sense in the soledad context.